src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/FileSystem.vb

```diff
@@ -0,0 +1,17 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System
+
+Namespace Microsoft.VisualBasic
+
+    Friend Module FileSystem
+
+        Friend Sub CloseAllFiles(ByVal assem As System.Reflection.Assembly)
+            ' CloseAllFiles(ProjectData.GetProjectData().GetAssemblyData(assem))
```

Comment:
Do you need an issue to complete this?

Reply:
[https://github.com/dotnet/corefx/issues/37091](https://github.com/dotnet/corefx/issues/37091)

---
In reply to: [277343922](https://github.com/dotnet/corefx/pull/37085#discussion_r277343922) [](ancestors = 277343922)